templates/etc/admin/admin.conf.erb

```diff
@@ -0,0 +1,87 @@
+# The pulp server configuration
```

Comment:
I tend to prefer that templates just exist at the top level of the folder, i.e. `templates/admin.conf.erb`. I know we have some hold over from when this was refactored that still has some files under the "path" it would exist, but since this can change and there isn't an overload of templates, I'd prefer they sit under templates/